operator/pkg/client/kgo_logging.go

```diff
@@ -10,27 +10,31 @@
 package client
 
 import (
+	"errors"
```

Comment:
NIT: We should have linter that prevents errors package and require `github.com/cockroachdb/errors` package

Reply:
I know that's the general use, but as this is a static error, not sure if we'd get much from the addition of a stacktrace there.